src/applications/yellow-ribbon/containers/SearchResults/index.jsx

```diff
@@ -31,14 +36,25 @@ export class SearchResults extends Component {
     ),
     page: PropTypes.number.isRequired,
     perPage: PropTypes.number.isRequired,
+    showYellowRibbonEnhancements: PropTypes.bool,
     totalResults: PropTypes.number,
   };
 
-  componentDidUpdate(prevProps) {
+  constructor(props) {
+    super(props);
+    this.state = {
+      isSearchResultsTipOpen: true,
+    };
+  }
+
+  componentDidUpdate(prevProps, prevState) {
     const justRefreshed = prevProps.fetching && !this.props.fetching;
 
     if (justRefreshed) {
       focusElement('[data-display-results-header]');
+      if (!prevState.isSearchResultsTipOpen) {
```

Comment:
This code makes me nervous tbh since `toggleAlertBoxAppearance` sets `isSearchResultsTipOpen` state in a componentDidUpdate, in future tweaks it'd be easy to accidentally cause an infinite loop here. So if I understand correctly we want a side-effect where after we're done fetching we toggle the alert box appearance assuming the search results tip isn't open. Ah ok, I see now that it's protected against the `justRefreshed` if statement.

Hmm, any reason we couldn't move `isSearchResultsTipOpen` into Redux, that way the side effect could happen in the thunk where we stop fetching? That way we don't have a side effect here that could potentially cause an infinite rerender if `fetching` is updated in the future